starter/script.js

```diff
@@ -1 +1,73 @@
-// put code here!
+var election = 0;
+var netLove;
+var cashMoney;
+var fightOrFlight;
+var cashOrVotes;
```

Comment:
can you think of a way to write this program without using more than one global variable?